projects/observability/src/shared/components/cartesian/chart-interactivty.ts

```diff
@@ -8,9 +8,16 @@ export const enum ChartEvent {
   Hover
 }
 
-export type ChartEventListener<TData> = (data: MouseLocationData<TData, Series<TData> | Band<TData>>[]) => void;
+export type ChartEventListener<TData> = (
+  data: MouseLocationData<TData, Series<TData> | Band<TData>>[] | CartesianHoverData<TData>
+) => void;
 
 export interface ChartTooltipTrackingOptions {
   followSingleAxis?: AxisType;
   radius?: number;
 }
+
+export interface CartesianHoverData<TData> {
+  groupId?: string;
```

Comment:
This isn't relevant to the hovering, just the syncing - make this a separate arg to the call on chart sync service (that way we can guarantee it's present)